expression/builtin_miscellaneous.go

```diff
@@ -92,6 +93,11 @@ var (
 	_ builtinFunc = &builtinNameConstDurationSig{}
 	_ builtinFunc = &builtinNameConstStringSig{}
 	_ builtinFunc = &builtinNameConstJSONSig{}
+	_ builtinFunc = &builtinTidbShardSig{}
+)
+
+const (
+	tidbShardBucketCount = 256
```

Comment:
What if we want to change it?

Reply:
We don't change it, 256 buckets are enough